Ideas-2021.md

```diff
@@ -0,0 +1,71 @@
+# GSoC 2021 Ideas
+
+[Administrative notes](#administrative-notes)
+
+------------
+# Administrative notes
+
+Above are a list of ideas we've planned for GSoC 2020 projects.
```

Comment:
GSoC 2020 should be changed to GSoC 2021

Reply:
Thanks, fixed in 821923e.